recipes/meme/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+MEME_ETC_DIR=${PREFIX}/etc
+cpanm YAML
+cpanm HTML::PullParser
```

Comment:
Sure @bgruening. I would want to modify the dependencies so they are packaged as well. However I am going to merge this for now and keep iterating.